rvic/core/param_file.py

```diff
@@ -49,7 +49,7 @@ def finish_params(outlets, dom_data, config_dict, directories):
     # subset (shorten time base)
     if options['SUBSET_DAYS'] and \
             options['SUBSET_DAYS'] < routing['BASIN_FLOWDAYS']:
-        subset_length = (options['SUBSET_DAYS'] *
+        subset_length = int(options['SUBSET_DAYS'] *
                          SECSPERDAY / routing['OUTPUT_INTERVAL'])
```

Comment:
We should make sure `SUBSET_DAYS` and `OUTPUT_INTERVAL` are both ints in the `init`call.